incubator/bootstrap_cli/__main__.py

```diff
@@ -202,14 +213,16 @@ class BootstrapCore:
 
     # Mark all auto-generated CDF Group names
     GROUP_NAME_PREFIX = "cdf:"
+    # TODO: make configurable and switch to 'all' as default
+    AGGREGATED_GROUP_NAME = "allprojects"
 
     def __init__(self, configpath: str, delete: bool = False):
         if delete:
             self.config: BootstrapDeleteConfig = BootstrapDeleteConfig.from_yaml(configpath)
             self.delete_or_deprecate: Dict[str, Any] = self.config.delete_or_deprecate
         else:
             self.config: BootstrapDeployConfig = BootstrapDeployConfig.from_yaml(configpath)
-            self.group_types_dimensions: Dict[str, Any] = self.config.bootstrap
+            self.group_bootstrap_dimensions: Dict[str, Any] = self.config.bootstrap
```

Comment:
This naming is still unclear to me. Can we come up with a better name?

Reply:
would love to, but atm this is 1:1 the `bootstrap` config part from yaml
- as such it represents the namespace>core "dimensions"
- maybe `group_**_hierarchy` reflects better the topology of the "dimension"?

Reply:
with `group_**_hierarchy` I would directly understand what is meant, but that might be just me

Reply:
agreed and changed